CHANGELOG.md

```diff
@@ -14,6 +14,9 @@
 
 [All changes in [[UnreleasedVersion]]](https://github.com/mozilla/uniffi-rs/compare/v0.23.0...HEAD).
 
+### ⚠️ Breaking Changes ⚠️
+- Implemented a new callback-interface ABI that signifacantly improves performance on Python and Kotlin.
```

Comment:
typo: signifacantly->significantly

Also wonder if we should call out that it's not breaking in a "break your code" sense, but most certainly breaking if you happen to maintain external bindings. On that note, there doesn't seem to be much guidance or even comments which might help these people update to use this scheme. I don't really want to block you on that though, so maybe you could open an issue which asks for such guidance to be given "soon", and link to that here?

Reply:
I expanded this comment with more details.  I don't think users need to do anything other than upgrade to the new version for this one.